README.md

```diff
@@ -69,15 +59,8 @@ I would like to receive a text such as "Thank you! Your order was placed and wil
 * Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
 
 
-In code review we'll be hoping to see:
-
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this at this moment.
-
-Notes on Test Coverage
-------------------
+Progress 
+--------
+I have implemented the first three of the four user stories.
```

Comment:
https://github.com/makersacademy/course/blob/master/pills/levels_of_stubbing.md is a pill about mocking 3rd party APIs 🙂